beautyjoy/BJC-Teacher-Tracker
# 187167707 teacher form submission

### [Pivotal Tracker Link][tracker]


[tracker]: https://www.pivotaltracker.com/story/show/your-story-id

## What this PR does:


Sends an email to the teacher (in addition to the admin) whenever they fill out a form. The new corresponding email template title is "Teacher Form Submission."

This pull request fixes|implements (pick one...) ______.



### Include screenshots, videos, etc.

#### Who authored this PR?

Arush




### How should this PR be tested?

Look at the RSpec and Cucumber tests to confirm that the feature works as intended.

* Is there a deploy we can view?
* What do the specs/features test?
* Are there edge cases to watch out for?

#### Are there any complications to deploying this?



Run "rails db:seed".

### Checklist:
